src/interface/definitions.yaml

```diff
@@ -50,13 +50,26 @@ Consent:
       allOf:
         - $ref: '#/definitions/FspId'
       description: PISP identifier who uses this Consent
+    status: 
+      $ref: '#/definitions/ConsentStatusEnum'
     scopes:
       type: array
       items:
         $ref: '#/definitions/Scope'
     credential:
       $ref: '#/definitions/Credential'
 
+ConsentStatusEnum:
+  title: ConsentStatus
+  type: string
+  enum:
+    - ACTIVE
+    - REVOKED
+  description: |
+    The status of the Credential's creation
+    - "ACTIVE" - The Consent is valid, and ready to be used by the PISP
+    - "REVOKED" - The Consent is no longer valid and has been requested to be deleted
```

Comment:
Maybe `The Consent is no longer valid and has been revoked` is better?

Reply:
makes sense